src/webui/www/private/css/style.css

```diff
@@ -461,6 +461,11 @@ a.propButton img {
 #torrentsFilterToolbar {
     display: inline-block;
     vertical-align: top;
+
+}
+
+#torrentsFilterToolbar.invisible {
+    display: none;
```

Comment:
Note that there's already code to hide add the `invisible` class to this element, but it's `display: none` rule was being overruled by the style defined directly against `#torrentsFilterToolbar`. The other option was the make the `invisible` class's `display` property `!important`, but I figured this could have broader implications.